controllers/articles-controller.js

```diff
@@ -7,24 +7,29 @@ const { selectArticles, selectArticleById, selectArticleCommentsById, updateArti
 // GET request controllers
 
 exports.getArticles = (req, res, next) => {
-  selectArticles().then((articles) => {
+  selectArticles()
+  .then((articles) => {
     res.status(200).send(articles)
   })
-}
+  .catch(next);
+};
 
 exports.getArticleById = (req, res, next) => {
   const { article_id } = req.params;
-  selectArticleById(article_id).then((article) => {
-      res.status(200).send(article.rows[0]);
-  });
+  selectArticleById(article_id)
+    .then((article) => {
+      res.status(200).send(article[0])
```

Comment:
Make sure you send an object with an `article` key in it (ie. `{ article:  {individual article object here} }`. You will have to change the way you access this in your tests as well.